nbformat/v4/nbformat.v4.schema.json

```diff
@@ -351,16 +351,14 @@
             "mimebundle": {
                 "description": "A mime-type keyed dictionary of data",
                 "type": "object",
-                "additionalProperties": false,
-                "properties": {
-                    "application/json": {
-                        "type": "object"
-                    }
+                "additionalProperties": {
+                  "description": "mimetype output (e.g. text/plain), represented as either an array of strings or a string.",
+                  "$ref": "#/definitions/misc/multiline_string"
                 },
                 "patternProperties": {
-                    "^(?!application/json$)[a-zA-Z0-9]+/[a-zA-Z0-9\\-\\+\\.]+$": {
-                        "description": "mimetype output (e.g. text/plain), represented as either an array of strings or a string.",
-                        "$ref": "#/definitions/misc/multiline_string"
+                    "^application/([a-zA-Z0-9.]+\\+)?json$": {
+                        "description": "Mimetypes with JSON output, can be an object",
+                        "type": "object"
```

Comment:
In reality I think this should be able to be any value as declared by the JSON spec. If we want to be conservative in interpreting RFC 4627, RFC 7159, and ECMA-404 the common ground here would be

```
  "type": ["object", "array"]
```

While technically it could be null, boolean, object, array, string, or number.